modules/library/main/src/main/java/org/geotools/util/CommonsConverterFactory.java

```diff
@@ -220,8 +220,8 @@
     }
     
     static class DateConverter implements Converter {
-        private static SimpleDateFormat format1 = new SimpleDateFormat( "yyyy-MM-dd HH:mm:ss.S a" );
-        private static SimpleDateFormat format2 = new SimpleDateFormat( "yyyy-MM-dd HH:mm:ssa" );
+        private SimpleDateFormat format1 = new SimpleDateFormat( "yyyy-MM-dd HH:mm:ss.S a" );
```

Comment:
Hum... interesting, the converters are created per conversion, or are always there?